_posts/2019-03-30-build-js-library-with-webpack-ringcentral-c2d.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "How to build a JS library with Webpack -- RingCentral C2D library"
+date:   2019-03-30 10:00:00 +0800
+categories: blog
+author: Embbnux Ji
+---
+
+![clicktodial](https://user-images.githubusercontent.com/7036536/51652788-d2627200-1fcb-11e9-8ba3-9e50baeaf8a6.png)
+
+Rencently, we publish RingCentral C2D library to help developers integrate RingCentral Click To Dial in browser. This library is built with Webpack 4 and Typescript. This article will share how to build a JS library which support CDN and NPM ways by introducing how we build RingCentral C2D.
```

Comment:
`Recently`, we `published the` RingCentral C2D library to hepl developers integration RingCentral `Click-to-Dial` in `browsers`. This library is built with Webpack 4 and Typescript.
This article `demonstrates` how to build a JS library that supports both CDN and NPM by introducing how we `built the` RingCentral C2D `library`.